### DISCUSSION TITLE
no such index [swnine_service_sla]

### DISCUSSION THREAD

**Discussion by why11xulei** [apache/skywalking#10572] [Q&A]
my project originally used skywalking v8.2,now it upgraded to v9.3,when i query rest api like unsuccessfulservice,this problem occurred.
Elasticsearch exception [type=index_not_found_exception, reason=no such index [swnine_service_sla]]
(swnine is sw_namespace)
I wonder if switch to another index.
The existing indexes are as follows：swnine_records-all，swnine_zipkin_span，swnine_log，swnine_metrics-all，swnine_browser_error_log，swnine_ui_template，swnine_segment

**Comment by wu-sheng** [comment]
You need to clean the indicies and templates in the ElasticSearch. Many things changed.

**Reply by why11xulei** [reply]
i start a new es docker and new mount directory,do i still need to clean the indicies and templates?

**Reply by wu-sheng** [reply]
I can't guide you on your private env. Generally, new env requires no manual setup.
